tests/_bases.py

```diff
@@ -13,10 +15,29 @@
 from tests import _helpers
 
 
-class TestEmbedBase(ABC, unittest.TestCase):
-    """Abstract base to provide helpers and fixtures."""
+class TestBase(unittest.TestCase):
+    """Base class for all test classes in the project."""
```

Comment:
The diff (especially in unified view) is a bit confusing here: I'm not actually renaming `TestEmbedBase`.